src/config.tsx

```diff
@@ -21,11 +21,19 @@
 import { getOSVersionFromNvr } from './utils/artifactUtils';
 import { ArtifactType } from './artifact';
 
-/*
- *  CI Dashboard configuration file
+/**
+ * CI Dashboard global configuration.
  */
 export const config = {
     defaultTitle: 'CI Dashboard',
+    graphQlServerUrl:
+        /*
+         * By default, we assume the GraphQL server endpoint is located at `/graphql`
+         * on the same server as the frontend. For local development, one may use
+         * the internal development instance of the GraphQL server. This URL can be
+         * overriden using the `REACT_APP_GRAPHQL_SERVER_URL` environment variable.
+         */
+        process.env.REACT_APP_GRAPHQL_SERVER_URL ?? '/graphql',
```

Comment:
Hi, I like this improvement.

Could you please update this PR with an example at readme file how to run this app?

Simple example + explanation: if you what to connect to local server during development run with

```
REACT_APP_GRAPHQL_SERVER_URL="http://localhost:5000/graphql" npm start
```

Reply:
Good idea. I've updated the readme.